### PR TITLE
nicer writegood-mode faces

### PR DESCRIPTION
**Pull request by dcherian**
Nicer faces for writegood-mode.

### CODE REVIEW

File: solarized-definitions.el [hunk 1]
```diff
@@ -215,6 +215,8 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
           ;; FIXME: don’t hardcode the SRGB color names
           (fmt-curl-red    `(                        :underline (:color "#dc322f" :style wave)))
           (fmt-curl-yellow `(                        :underline (:color "#b58900" :style wave)))
+          (fmt-curl-magenta `(                      :underline (:color "#d33682" :style wave)))
+          (fmt-curl-cyan `(                      :underline (:color "#2aa198" :style wave)))
```
Comment (sellout) [inline solarized-definitions.el]: This looks good. Just some stylistic requests.

Could you line up `:underline` with the other ones?

File: solarized-definitions.el [hunk 2]
```diff
@@ -700,6 +702,10 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
                 (whitespace-trailing (,@fmt-bold ,@fg-red ,@bg-base02))
                 (whitespace-highlight-face (,@fg-red ,@bg-blue))
                 (whitespace-line (,@fg-magenta))
+                ;; writegood
+                (writegood-weasels-face (,@fmt-curl-cyan,@fg-cyan))
```
Comment (sellout) [inline solarized-definitions.el]: And add a space before the second `,` here?

Thanks!

**Comment by dcherian** [comment]
Updated, not sure why CI fails.

**Comment by sellout** [comment]
Thanks!

**Pull request merged**